managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py

```diff
@@ -0,0 +1,147 @@
+import boto3
+import logging
+import uuid
+import jwt
+import base64
+
+region='us-east-1'
+
+# boto3.set_stream_logger(name='botocore',level=10)
+
+def create_license(SignKey):
+    """Creates a license.
+
+    For the purposes of this sample, it creates a dummy license only allowing
+    to specify the SignKey to be used to signed the response of borrow.
+
+    :SignKey KMS Asymmetric Key Arn
+    """
+
+    lm_client = boto3.client('license-manager', region)
+    return lm_client.create_license(
+        LicenseName="My License",
+        ProductName="My Product",
+        Beneficiary="My Beneficiary",
+        ConsumptionConfiguration={
+            "ProvisionalConfiguration": {
+                "MaxTimeToLiveInMinutes": 60
+            },
+            "BorrowConfiguration": {
+                "MaxTimeToLiveInMinutes": 1440,
+                "AllowEarlyCheckIn": True
+            },
+        },
+        Entitlements=[{
+            "Name":"Users",
+            "Unit":"Count",
+            "MaxCount": 1000,
+            "Overage": False,
+            "AllowCheckIn": True
+        }],
+        HomeRegion=region,
+        Issuer={
+            "Name":"My Company",
+            "SignKey": SignKey
+        },
+        ProductSKU="3688750d24200a4daa2e046732580b30",
+        Validity={
+            "Begin": "2021-04-12T00:00:00Z"
+        },
+        LicenseMetadata=[{
+            "Name":"ProductName",
+            "Value":"My awesome product"
+        }],
+        ClientToken=str(uuid.uuid4())
+    )
+
+def checkout_borrow_license(LicenseArn):
```

Comment:
Lets add some explanation on when we use the API.
Also some notes on if the token expires or not for checked out borrow tokens.

Reply:
Added in read me